tests/integration/conftest.py

```diff
@@ -10,6 +11,42 @@
 from yaptide.persistence.database import db
 
 
+@pytest.fixture(scope='session')
+def small_simulation_payload(payload_editor_dict_data : dict) -> Generator[dict, None, None]:
+    """Small simulation payload for testing purposes"""
+    payload_dict = copy.deepcopy(payload_editor_dict_data)
+
+    # limit the particle numbers to get faster results
+    payload_dict["ntasks"] = 2
+    payload_dict["input_json"]["beam"]["numberOfParticles"] = 12
+
+    # reduce number of segments to get results which are easier to print
+    for detector in payload_dict["input_json"]["detectorManager"]["detectors"]:
+        for segments_type in ["xSegments", "ySegments", "zSegments"]:
+            # if segments_type is missing (for example xSegments is missing for cylinder detector)
+            # then we skip it by comparing default value 2 > 1
+            if detector["geometryData"]["parameters"].get(segments_type, 2) > 1:
+                detector["geometryData"]["parameters"][segments_type] = 4
```

Comment:
I am not sure if I understand this correctly it seems that we are setting the value to 4 if:

- detector["geometryData"]["parameters"][segments_type] is not defined
- detector["geometryData"]["parameters"][segments_type] is less than or equal to 1

To update only existing values we can use
```
if segments_type in detector["geometryData"]["parameters"]
```

Reply:
I would like to update that value if one of following condition holds:
 - `segments_type`  key doesn't exists
 - `segments_type`  key does exists and corresponding value is greater than one

Reply:
Right, 
Ok. So the logic is correct, I was just misguided by the comment above:

```
# then we skip it by comparing default value 2 > 1
```

Reply:
I could rephrase the comment, in fact the code is a bit tricky. Is segment key is missing, then a default value 2 is retrieved. Then a comparison 2 > 1 is evaluated which is true.